### PR TITLE
v0.17.2

### PR DESCRIPTION
**Pull request by Cmancuso**
v0.17.2 August 5, 2022

- Fixes
     - Updated concat_columns to work with dataframes with mismatched indices or different shapes (#485)
- Documentation Changes
     - Add instructions to add new users to woodwork feedstock (#483)
- Testing Changes
     - Add create feedstock PR workflow (#1489)

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -3,18 +3,25 @@
...
-        * Updated concat_columns to work with dataframes with mismatched indices or different shapes (:pr:`1485`)
     * Changes
     * Documentation Changes
-       * Add instructions to add new users to woodwork feedstock (:pr:`1483`)
     * Testing Changes
 
+.. Thanks to the following people for contributing to this release:
+
+v0.17.2 August 5, 2022
+==============
```
Comment (gsheni) [inline docs/source/release_notes.rst:17]:
```suggestion
v0.17.2 August 5, 2022
=================
```

File: woodwork/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "0.17.1"
+__version__ = "0.17.2"
```
Comment (gsheni) [inline woodwork/version.py:1]: You will need to update `test_version.py`

**Review by bchen1116 — APPROVED** [review]
🚢

**Review by ParthivNaresh — APPROVED** [review]
Looks solid to me thanks!

**Pull request merged**